enthought_example/version.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.3.0.dev0'
```

Comment:
Should probably be `0.4.0.dev0` now?

Reply:
There hasn't been a 0.3 release for this plugin yet, it was unclear to me whether plugin <-> bdss compatibility should be tracked via `version.py` or if the `PLUGIN_VERSION` (integer!) attribute in `example_plugin.py` should be used instead.
For me `__version__` should be a devlopment thing independent of the bdss versions and `PLUGIN_VERSION` should be renamed and refer to the  min(/max) compatible version of the bdss.

Reply:
Well I guess you already set the bdss compatibility stuff in setup.py